lib/util/git/modified-cache.ts

```diff
@@ -17,7 +19,6 @@ export function getCachedModifiedResult(
 
 export function setCachedModifiedResult(
   branchName: string,
-  branchSha: string,
```

Comment:
If this is missing from the params then shouldn't it still be written out? Or you assume it's already written out?

Reply:
Not sure what written out means?